<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboColumn">
       <property name="toolTip">
        <string>Field to be searched</string>
       </property>
       <property name="whatsThis">
        <string>Use this control to select the field to be searched in the current table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboOperator">
       <property name="toolTip">
        <string>Search criteria: use 'contains' for partial matches</string>
       </property>
       <property name="whatsThis">
        <string>This control is used to select the search criteria used to look for the search term in the database. Use '=' or 'contains' to find words, and the comparison symbols to filter numeric data.</string>
       </property>
       <item>
        <property name="text">
         <string>=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>contains</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;&gt;</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="editSearchString">
       <property name="toolTip">
        <string>Enter values or words to search</string>
       </property>
       <property name="whatsThis">
        <string>This is a place to enter the word or number to be searched in the database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSearch">
       <property name="toolTip">
        <string>Perform the search</string>
       </property>
       <property name="whatsThis">
        <string>This button starts the search process</string>
       </property>
       <property name="text">
        <string>&amp;Search</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableResults">
     <property name="whatsThis">
      <string>Results of the search will appear in this area. Click on a result to select the corresponding record in the database</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>Records</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNumberResults">
     <property name="text">
      <string>Found: 0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboColumn</tabstop>
  <tabstop>comboOperator</tabstop>
  <tabstop>editSearchString</tabstop>
  <tabstop>buttonSearch</tabstop>
  <tabstop>tableResults</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSearch</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>find()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableResults</sender>
   <signal>itemClicked(QTableWidgetItem*)</signal>
   <receiver>FindDialog</receiver>
   <slot>recordSelected(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>find()</slot>
  <slot>recordSelected(QTableWidgetItem*)</slot>
 </slots>
</ui>
